nevergrad/functions/gym/multigym.py

```diff
@@ -315,6 +315,12 @@ def get_env_names() -> tp.List[str]:
             try:
                 assert "Kelly" not in str(e.id)  # We should have another check than that.
                 assert "llvm" not in str(e.id)  # We should have another check than that.
+                assert "BulletEnv" not in str(e.id)  # We should have another check than that.
+                assert "Minitaur" not in str(e.id)  # We should have another check than that.
+                assert "Kuka" not in str(e.id)  # We should have another check than that.
```

Comment:
can all thise be factorized? assert str(e.id) not in....

Reply:
Not sure to see how because I need to match part of the string.
I could do a regex to factories, but not sure that it would be clearer.

Reply:
no, regex are a mess :D just : 
```python
assert not any(x in str(e.id) for x in "Kelly llvm BulletEnv Minitaur Kuka".split())
```